data/mods/My_Sweet_Cataclysm/sweet_monsters.json

```
@@ -178,7 +178,7 @@
     "id": "mon_marshmallow_buff_armored",
     "type": "MONSTER",
     "name": { "str": "marshmallow champion" },
-    "description": "Standing tall in its shining armor of chocolate coated crakers this marshmallow is proudly striding towards an unknown goal.",
+    "description": "Standing tall in its magnificent shining armor of chocolate-coated crackers, this marshmallow proudly strides towards an unknown goal.",
```

```suggestion
    "description": "Standing tall in its shining armor of chocolate-coated crackers, this marshmallow proudly strides towards an unknown goal.",
```
Magnificent is a bit too much I think